README.md

```diff
@@ -1,221 +1,139 @@
-    <td><a href="https://photos.google.com/">Google Photos</a></td>
-    <th>On</th>
-  </tr>
-  <tr>
-    <td><code>dropbox</code></td>
-    <td><a href="https://www.dropbox.com/">Dropbox</a></td>
-    <th>On</th>
-  </tr>
-  <tr>
-    <td><code>instagram</code></td>
-    <td><a href="http://instagram.com/">Instagram</a></td>
-    <th>On</th>
-  </tr>
-  <tr>
-    <td><code>evernote</code></td>
-    <td><a href="http://evernote.com/">Evernote</a></td>
-    <th>On</th>
-  </tr>
-  <tr>
-    <td><code>flickr</code></td>
-    <td><a href="https://www.flickr.com/">Flickr</a></td>
-    <th>On</th>
-  </tr>
-  <tr>
-    <td><code>skydrive</code></td>
-    <td><a href="https://onedrive.live.com/">OneDrive</a></td>
-    <th>On</th>
-  </tr>
-  <tr>
-    <td><code>box</code></td>
-    <td><a href="https://www.box.com/">Box</a></td>
-    <th>Off</th>
-  </tr>
-  <tr>
-    <td><code>vk</code></td>
-    <td><a href="http://vk.com/">VK</a></td>
-    <th>Off</th>
-  </tr>
-  <tr>
-    <td><code>huddle</code></td>
-    <td><a href="https://www.huddle.com/">Huddle</a></td>
-    <th>Off</th>
-  </tr>
-</table>
-
-The set of enabled sources can be reconfigured.
-This is done through specifying their respective 
-codes in an options string, as space-separated values.
-Use the `all` value to enable all supported sources.
-
-```html
-<script type="text/javascript">
-  UPLOADCARE_TABS = 'file url instagram flickr';
-</script>
+## Configuration
+
+### Plugin configuration
+
+To apply a custom configuration, initialize the TinyMCE editor
+providing additional options:
+
+```javascript
+tinymce.init({
+  selector: '#editor',
+  plugins: 'uploadcare',
+  toolbar: 'uploadcare',
+  external_plugins: {
+    uploadcare: '/path/to/uploadcare/plugin.js',
+  },
+  uploadcare_public_key: 'YOUR_PUBLIC_KEY',
+  uploadcare_image_shrink: '500x375', // set client-size resize for images
+  uploadcare_multiple: true, // allow multi-file uploads
+  uploadcare_multiple_max: 3,
+  uploadcare_crop: '1:1,4:3', // set crop options when handling images
+  /* feel free to add more options here */
+})
 ```
 
-## Other settings
-
-All the Uploadcare Widget settings can be found in our [docs][4].
-Please read those to unleash the uploading power in its full.
-
-# Contributors
-
-* [@grayhound](https://github.com/grayhound)
-* [@dmitry-mukhin](https://github.com/dmitry-mukhin)
-* [@dimaninc](https://github.com/dimaninc)
-* [@Zmoki](https://github.com/Zmoki)
-
-# Contact
-
-Got any thoughts to share? Hit us up at
-[hello@uploadcare.com](mailto:hello@uploadcare.com).
-
-[1]: https://uploadcare.com/
-[2]: http://www.tinymce.com/
-[3]: https://uploadcare.com/documentation/widget/#crop
-[4]: https://uploadcare.com/documentation/widget/#configuration
+You can use any [widget options][uc-docs-widget-options]
+that are allowed to use as object key. Just add the `uploadcare_` prefix and
+use snake_case instead of camelCase for a name of an option, e.g.
+`imagesOnly` &rarr; `uploadcare_images_only`.
```

Comment:
... widget options providing those as object keys.
Just add the `uploadcare_` prefix and use `snake_case` instead of `camelCase` in option names, e.g., ...